php/fpm/php-fpm.conf

```diff
@@ -16,7 +16,7 @@ pm.max_requests      = ${PHP_FPM_MAX_REQUESTS}
 
 pm.status_path = /status
 
-access.format='{ "timestamp": "%{%Y-%m-%dT%H:%M:%S%z}T", "fields": { "client_ip": "%{HTTP_X_FORWARDED_FOR}e", "remote_addr": "%R", "remote_user": "%u", "request_uri": "%{REQUEST_URI}e", "status": "%s", "body_bytes_sent": "%l", "request_time": "%d", "http_referrer": "%{HTTP_REFERER}e", "http_user_agent": "%{HTTP_USER_AGENT}e", "request_id": "%{HTTP_X_REQUEST_ID}e", "cpu": "%C", "memory": "%M" } }'
+access.format='{ "timestamp": "%{%Y-%m-%dT%H:%M:%S%z}T", "fields": { "client_ip": "%{HTTP_X_FORWARDED_FOR}e", "remote_addr": "%R", "remote_user": "%u", "request_uri": "%{REQUEST_URI}e", "status": "%s", "body_bytes_sent": "%l", "request_time": "%d", "http_referrer": "%{HTTP_REFERER}e", "http_user_agent": "%{HTTP_USER_AGENT}e", "request_id": "%{HTTP_X_REQUEST_ID}e", "cpu": "%C", "memory": "%M", "headers": { "Cache-Control": "%{Cache-Control}o"} } }'
```

Comment:
Why is there an `o` after the last brace?

Reply:
Don't really know the formatting...

Reply:
Source: https://www.php.net/manual/en/install.fpm.configuration.php

Scroll down to the table underneath `access.format`.

Example usage:
```
;  %o: output header
;      it must be associated with embraces to specify the name of the header:
;      - %{Content-Type}o
;      - %{X-Powered-By}o
;      - %{Transfert-Encoding}o
```

Reply:
It's so that we can add more if we need to, without overriding the value of `%o` - so it's `%{header name}o`.